fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1706,7 +1706,10 @@ void SimulationConfig::setEncryptionAtRestMode(const TestConfig& testConfig) {
 		// If encryptModes are not specified, give encryption higher chance to be enabled.
 		// The good thing is testing with encryption on doesn't loss test coverage for most of the other features.
 		available = std::vector<bool>(EncryptionAtRestMode::END, true);
-		probability = { 0.25, 0.5, 0.25 };
+		// Enabling encryption require the use of Redwood storage engine, but we don't want to test with Redwood with
```

Comment:
am not sure if "we don't want to test Redwood in simulation" seems generic statement from an open-source repo, point of view, Redwood might be storage engine of choice of some organizations, at the same time, others may have reasons to choose otherwise :)